.github/workflows/publish.yml

```diff
@@ -1,10 +1,7 @@
 name: publish
 
 on:
-  workflow_dispatch:
```

Comment:
[optional] consider leaving this in if it wasn't causing issues. This allowed us to test PRs to the publish workflow. With it, you can go to workflow in the Actions tab and run the workflow on the PR.